NeuroID/src/test/java/com/neuroid/tracker/service/NIDValidationServiceTest.kt

```diff
@@ -100,4 +100,26 @@ class NIDValidationServiceTest {
 
         Assert.assertEquals(false, value)
     }
+
+    // scrubIdentifier
+    @Test
+    fun test_scrubIdentifier_email() {
+        val value = validationService.scrubIdentifier("test@test.com")
+
+        Assert.assertEquals("t**t@test.com", value)
+    }
+
+    @Test
+    fun test_scrubIdentifier_ssn() {
+        val value = validationService.scrubIdentifier("123-45-6789")
+
+        Assert.assertEquals("***-**-6789", value)
+    }
```

Comment:
Just curious- Should we scrub the last 4 digits of Ssn and leave the rest because that's more unique than the rest?